release/src/Main.hs

```diff
@@ -86,6 +87,14 @@ main = do
       uploadArtifacts <- concatMapM (mavenArtifactCoords optsAllArtifacts) mavenUploadArtifacts
       validateMavenArtifacts releaseDir uploadArtifacts
 
+      -- npm packages we want to publish.
+      let npmPackages =
+            [ "//language-support/ts/daml-ledger"
```

Comment:
We might need to upload these packages in reverse order since the former depends on the latter. I suggest we reverse the list to be on the safe side.

Reply:
I'll change it to be on the safe side.